sootup.callgraph/src/main/java/sootup/callgraph/GraphBasedCallGraph.java

```diff
@@ -129,6 +133,93 @@ public int callCount() {
     return graph.edgeSet().size();
   }
 
+  @Override
+  public String exportAsDot() {
+    StringBuilder dotFormatBuilder = new StringBuilder();
+
+    graph.edgeSet().stream()
+        .sorted(
+            Comparator.comparing(
+                    (Edge edge) -> graph.getEdgeSource(edge).toString(), Comparator.naturalOrder())
```

Comment:
dont sort the entire signature.
otherwise the order will be class name , return type, method name and then parameter.
but we dont want that the return type highly effects the order. So you have to compare the parts of the method signatures